src/decorator/columns/ViewColumn.ts

```diff
@@ -0,0 +1,21 @@
+import {getMetadataArgsStorage} from "../../";
+import {ColumnMetadataArgs} from "../../metadata-args/ColumnMetadataArgs";
+
+/**
+ * ViewColumn decorator is used to mark a specific class property as a view column.
+ */
+export function ViewColumn(): Function;
```

Comment:
Why do we need separate definition here?

Reply:
not needed, just a mistake